packages/react-components/public/intro.css

```diff
@@ -58,6 +58,13 @@ h1.fluent {
   letter-spacing: -0.16px;
 }
 
+h1 .fluent-version {
+  display: block;
+  font-size: 24px; /* --font-size-base-600 */
+  line-height: 32px;
+  color: #707070; /* --color-neutral-foreground-3 */
+}
```

Comment:
Added the actual version number to the intro heading in lieu of vNext:

![image](https://user-images.githubusercontent.com/5067638/152237338-775e6526-06ca-4fee-84ec-dc979297acb5.png)